dist/docs/guide/usage/add_feature_pack.adoc

```diff
@@ -0,0 +1,70 @@
+### Installing additional Galleon Feature Packs
+
+Galleon feature packs can be used to extend capabilities offered by base server and provide features like, for example, https://github.com/wildfly-extras/wildfly-datasources-galleon-pack[datasource support] or https://github.com/wildfly-extras/wildfly-myfaces-feature-pack[alternative JSF implemetnation].
+
+Prospero allows users to install such feature pack on top of existing server using `feature-pack add` operation.
+
+#### Feature pack channels
+
+As with all the components used to provision the server, Prospero relies on channels to supply the feature pack and any artifacts it may require. In some cases the base server channel may offer the additional feature packs, in others, an additional channel will have to be registered with the server. See <<Subscribing to a channel>> for information how to subscribe to a channel.
+
+Sometimes feature packs may require additional artifacts whose lifecycle is separate from the feature pack itself. For example, Datasources feature pack lets users specify which version of database driver should be installed irrespective of the feature pack version. In such cases users have to create and subscribe to a new channel providing those artifacts.
+
+For example, a channel supplying Datasources feature pack could use https://repo1.maven.org/maven2[Maven Central] repository and a following manifest:
+```
+schemaVersion: 1.0.0
+streams:
+  - groupId: org.wildfly
+    artifactId: wildfly-datasources-galleon-pack
+    version: 4.0.1.Final
+  - groupId: com.mysql
+    artifactId: mysql-connector-j
+    version: 8.1.0
+```
+
+#### Installing feature pack
+
+The feature packs are identified by GroupId:ArtifactId part of their Maven coordinates. This is known as their Feature Pack Location (fpl).
+
+For example, assuming the server is subscribed to a datasources channel, following command will install the datasources feature pack:
+
+```
+$ prospero feature-pack add \
+  --fpl org.wildfly:wildfly-datasources-galleon-pack \
+  --dir wildfly
+```
+
+#### Customizing installed feature packs
+
+The Galleon feature packs may provide different capabilities using https://docs.wildfly.org/galleon/#_layers[layers]. Users may select a set of layers to install using `--layers` parameter.
+
+If the feature pack changes server configuration files, the user may also choose which server configuration should be affected using `--model` and `--config` parameters.
```

Comment:
Maybe mention that the default model and usually the only provided model is "standalone" and the default config is "standalone.xml", if that's correct?

Reply:
Good idea, thanks!